spec.md

```diff
@@ -936,7 +976,9 @@ message NodePublishVolumeResponse {
 
 A Node Plugin MUST implement this RPC call.
 This RPC is a reverse operation of `NodePublishVolume`.
-If the corresponding Controller Plugin has `PUBLISH_UNPUBLISH_VOLUME` controller capability, the CO MUST guarantee that this RPC is called before `ControllerUnpublishVolume` is called for the given node and the given volume.
+This RPC MUST undo the work by the corresponding `NodePublishVolume`.
+This RPC SHOULD be called by the CO at least once for each `target_path` that was successfully setup via `NodePublishVolume`.
```

Comment:
is there a particular reason that this says SHOULD vs. SHALL? are there any valid use cases for a CO to successfully invoke `NodePublishVolume` prior to a workload .. and never call `NodeUnpublishVolume`, ever?